src/themes/default/components/Text/index.js

```diff
@@ -25,13 +27,16 @@ const enhance = compose(
  * @param {string} param.children - markdown-generated html to render.
  * @return {Object} React component
  */
-const Text = ({ title, children, isLocal }) => (
-    <Fade in={true} appear={true}>
-        <div
-            className={`Text ${isLocal ? 'is-local' : ''}`}
-            dangerouslySetInnerHTML={createMarkup(children, title)}
-        />
-    </Fade>
-);
+const Text = ({ title, children, isLocal }) =>
+    (title || children) && (
```

Comment:
will this ever be called without a title and children? cause then it will break because it doesn't return anything

Reply:
In this case it is being called without title and children. The point is not to return anything in that case.